buildSrc/src/main/java/org/elasticsearch/gradle/BwcVersions.java

```diff
@@ -328,15 +330,17 @@ public void compareToAuthoritative(List<Version> authoritativeReleasedVersions)
     }
 
     public List<Version> getIndexCompatible() {
-        return unmodifiableList(
-            Stream.concat(groupByMajor.get(currentVersion.getMajor() - 1).stream(), groupByMajor.get(currentVersion.getMajor()).stream())
-                .collect(Collectors.toList())
-        );
+        var collect = Stream.concat(
+            groupByMajor.get(currentVersion.getMajor() - 1).stream(),
+            groupByMajor.get(currentVersion.getMajor()).stream()
+        ).collect(Collectors.toList());
+        return Architecture.current() == Architecture.AARCH64
```

Comment:
nit: can we put a newline before the `return`.

Reply:
done